src/drivers/gpio/stm32/stm32_gpio_conf_l0x0.h

```diff
@@ -0,0 +1,86 @@
+/**
+ * @file
+ *
+ * @date 07.02.2021
+ * @author Vadim Deryabkin
+ */
+
+#ifndef SRC_DRIVERS_GPIO_STM32_STM32_GPIO_CONF_L0X0_H_
+#define SRC_DRIVERS_GPIO_STM32_STM32_GPIO_CONF_L0X0_H_
+
+#include <hal/reg.h>
+
+#define PINS_NUMBER 16
+#define STM32_GPIO_PORTS_COUNT 6
+
+#define REG_RCC        0x40021000       // Doc: DocID031151 Rev 1, RM0451, 39/774.
+#define REG_RCC_IOPENR (REG_RCC + 0x2C) // Doc: DocID031151 Rev 1, RM0451, 169/774.
+
+typedef struct {
```

Comment:
Maybe extract similar structure declaration into a common header?
It seems that the same structure is declared in stm32_gpio_conf_f0.h

Reply:
It struct and that struct (for f0 and for l010) are different.

Reply:
Hm, It's strange. I see
In stm32_gpio_conf_f0.h
```
typedef struct {
	volatile uint32_t MODER;
	volatile uint32_t OTYPER;
	volatile uint32_t OSPEEDR;
	volatile uint32_t PUPDR;
	volatile uint32_t IDR;
	volatile uint32_t ODR;
	volatile uint32_t BSRR;
	volatile uint32_t LCKR;
	volatile uint32_t AFRL;
	volatile uint32_t AFRH;
	volatile uint32_t BRR;
} gpio_struct; // Doc: DocID025023 Rev 4, 143/779.
```

and in this one I see
```
typedef struct {
   volatile uint32_t MODER;   // 0x00
    volatile uint32_t OTYPER;  // 0x04
    volatile uint32_t OSPEEDR; // 0x08
    volatile uint32_t PUPDR;   // 0x0C
    volatile uint32_t IDR;     // 0x10
    volatile uint32_t ODR;     // 0x14
    volatile uint32_t BSRR;    // 0x18
    volatile uint32_t LCKR;    // 0x1C
    volatile uint32_t AFRL;    // 0x20
    volatile uint32_t AFRH;    // 0x24
    volatile uint32_t BRR;     // 0x28
} gpio_struct; // Doc: DocID031151 Rev 1, RM0451, 205/774.
```
It seems that the fields are equivalent

Reply:
Hmm... Okay. This I will add it.